resources/mapsed-master/examples/06-full-example.js

```diff
@@ -0,0 +1,324 @@
+			else if (markerType == "new")
+				// user has clicked on the add place (+) icon to add a new place
+				imageUrl = "examples/images/add-place.png";
+			else 
+				// normal Google Places result
+				imageUrl = "examples/images/google-place.png";
+				
+			return {
+				url: imageUrl,
+				size: new google.maps.Size(28, 40),
+				origin: new google.maps.Point(0, 0),
+				
+				// where the little cross-hair appears (on new markers) relative to the image
+				anchor: new google.maps.Point(14, 45)
+			};
+		},
+				
+		// shows additional instructions to the user
+		getHelpWindow: function(m) {
+			var html = 
+				"<div class='mapsed-help'>" +
+					"<h3>Find a venue</h3>" +
+					"<ol>" +
+						"<li>Simply use the <strong>search</strong> box to find a venue in your area.</li>" +
+						"<li>On the pop-up, click <strong>Select</strong> to pick a pitch.</li>" + 
+					"</ol>" +
+					"<h3>New venues</h3>" +
+					"<ol>" +
+						"<li>Your venue isn't displayed?  Simply click on the map where your pitch is.</li>" +
+						"<li>Fill in the details in the dialog.</li>" + 
+						"<li>You can drag the marker around to pinpoint the right location.</li>" + 
+						"<li>Once you're happy, click the <strong>OK</strong> button</li>" + 
+					"</ol>" +
+				"</div>"
+			;
+
+			return html;
+		},
+		showHelpOnLoad: true,
+
+		// Callback for when the user tries to load the map
+		onClose: function(mapsed) {
+			var closeMap = confirm("Close map?");
+
+			// you can cancel the close of the map by returning false
+			return closeMap;
+		},
+		
+	    // Fired once the mapsed object has initialised, but before the map 
+		// has been drawn.  If you wish to add custom controls, this is where to do it
+		onPreInit: function(mapsed) {
+			var html = "",
+					$select = null,
+					$mapContainer = null
+			;
+			
+			// build up the theme picker
+			html += "<select id='themePicker' title='Pick an alternative map style...' class='mapsed-control-button'>";
+			for (var i=0; i < _snazzyMaps.length; i++) {
+				var theme = _snazzyMaps[i];
+				html += "<option value='" + theme.name + "'>" + theme.name + "</option>";
+			}
+			html += "</select>";
+
+			$select = mapsed.addMapControl(html, google.maps.ControlPosition.TOP_RIGHT);
+			
+			// wire up the change event to pick a new theme
+			$select.on("change", function() {
+			    onThemeChange($(this), mapsed);
+			});
+			
+			
+			// add warning about problems with POI being turn off with custom maps
+			html = 
+				"<div>" + 
+					"<p>Please note that POI cannot be turned off when using styled maps.</p>" + 
+				"</div>"
+			;
+			mapsed.addMapControl(html, google.maps.ControlPosition.BOTTOM_LEFT);
+		},
+		
+		// Fired once the map has completed loading
+		onInit: function(m) {
+			
+		}
+
+	});
+
+}
+
+$(document).ready(function() {			
+	// wire up examples
+	
+	// wire up full window example
+	$("#show-full-window")
+		.on("click", fullWindowExample)
+		//.trigger("click")
+	;
+	
+});
+
```

Comment:
Are all these examples from some plugin you pulled in? If so, generally they should be kept outside of the "production" environment of your site. Rule of thumb is to only sync functional code.